### PR TITLE
Add source name tag in log line

### PR DESCRIPTION
**Pull request by darko-poljak**
@telmich  Added tag to log lines so that source name is associated with log line.
With this the output is like the following (test0 is the source that is being backed up):
```
2018-09-20-16:09:19: ccollect 2.3: Beginning backup using interval est
2018-09-20-16:09:19: [test0] Beginning to backup
2018-09-20-16:09:19: Error: [test0] No definition for interval "est" found. Skipping.
2018-09-20-16:09:19: Finished
```

Notice normal and error log lines.

Do you have a proposal for nicer implementation though?

### CODE REVIEW

File: ccollect [hunk 1]
```diff
@@ -808,7 +809,7 @@ while [ "${source_no}" -lt "${no_sources}" ]; do
     minutes="$(((${full_seconds} % 3600) / 60))"
     seconds="$((${full_seconds} % 60))"
 
-    _techo "Backup lasted: ${hours}:${minutes}:${seconds} (h:m:s)"
+    _techo "${tag}" "Backup lasted: ${hours}:${minutes}:${seconds} (h:m:s)"
 
     unlock "${name}"
 ) | add_name
```
Comment (telmich) [inline ccollect]: Erm... confused. Isn't that what I wrote add_name for initially?

Comment (darko-poljak) [reply ccollect]: Hm... I overlooked this. Although this works only for stdout. I will adapt it to work with syslog and file logging too.

**Comment by darko-poljak** [comment]
@telmich Look at this change. This way logging to stdout, file, syslog, all will be tagged by _name_.

**Comment by telmich** [comment]
ok, good to go

**Pull request merged**